### ISSUE TITLE
Android App doesn't read tags from Pro version

### ISSUE THREAD

**Issue by tommyguitfiddle** [tagspaces/tagspaces#634]
The current android app (2.8.0) does not read the tags that I have created with the pro version on my desktop (Linux .deb).  I'm using the sidecar folder for tags, as that was the feature that most interested me in getting the pro version.  It makes the pro version basically useless to me if I can't access my to-do list on my mobile device.  I was under the impression that there would be read-access in the mobile app.  Is there a setting I'm missing, or is this just not possible yet?

**Comment by mjh7** [comment]
I have the same issue - this was also the reason I got the Pro version :( . Glad to see there's ongoing development on a new version for desktop at least. Is work on android on the roadmap? Is iOS app dead forever? I was initially excited to find a cross-platform solution but haven't gotten much use out of it.
Notes: if I search for a specific tag that I created it does find the associated file(s), but tags do not appear on the files nor in tag groups like they do in desktop version.
I've only tried pdfs so far but the "TagSpaces Lite" app isn't able to display them either.
